src/js/components/Fetch.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+const TIMEOUT = 10000; // 10 seconds
```

Comment:
Could you refactor it so that Fetch accepts timeout as a prop and defaults to 10s?

Reply:
Yup, for sure! 👍